### ISSUE TITLE
Duplicate change notes created when changing first publication date

### ISSUE THREAD

**Issue by chrisroos** [alphagov/manuals-publisher#970]
This is the bug that resulted in the problem with The Highway Code described in issue #861.

To replicate on integration:

1. Create a manual
2. Add a section
3. Publish the manual
4. View the manual on the website
5. Observe that the updates for the manual list a single change with "New section added"
6. In Manuals Publisher click "Edit first publication date"
7. Select "Change the first published date" and set the date and time to now
8. Click "Save as draft"
9. Click "Publish manual"
10. View the manual on the website
11. Observe that the updates for the manual now list two "New section added" changes.

We haven't investigated in detail but it would seem that the problem is caused by the most recent SectionEdition(s) being duplicated. If the SectionEdition is a major change then a corresponding PublicationLog will be created. The PublicationLogs are used to build the change notes for a manual.

**Comment by steventux** [comment]
The Publishing API event logs confirm that the change notes contain the duplicate `PublicationLog` entry when the second `PutContent` event is sent (example test data from integration):
```
:change_notes=>[{:base_path=>"/guidance/sl-gh-970/introduction", :title=>"Introduction", :change_note=>"New section added.", :published_at=>"2017-04-24T14:29:48Z"}, {:base_path=>"/guidance/sl-gh-970/introduction", :title=>"Introduction", :change_note=>"New section added.", :published_at=>"2017-04-24T14:55:36Z"}]
```

**Comment by chrisroos** [comment]
@steventux's fix in PR #1040 has now been deployed and I'm no longer able to replicate this problem on integration so I'm closing this issue.
